### PR TITLE
Feat/home

### PR DESCRIPTION
**Pull request by srishti-maurya**
### Added
- add new note
- delete a note
- edit a note
- toasts
- private routes

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+//components
+import { Nav } from "./frontend/components";
+//routes
 import { PageRoutes } from "./frontend/routes/PageRoutes";
 
 function App() {
   return (
     <div className="App">
       <Nav />
+      <Toaster position="bottom-right" />
```
Comment (KaustubhM1997) [inline src/App.js:13]: Suggestion: inline styles could be avoided

Comment (srishti-maurya) [reply src/App.js:13]: It is a prop to the `Toaster` component

**Review by KaustubhM1997 — APPROVED** [review]
Everything looks good to merge!

**Review by pritam-kr — APPROVED** [review]
Hey, Code looks clean-
- UI and UX are also good
- I have a question, how did you implement background and hero text going over the nav bar fix it.
Screenshot (49)

**Pull request merged**